woocommerce-services.php

```diff
@@ -843,7 +843,7 @@ public function rest_api_init() {
 			$this->set_rest_self_help_controller( $rest_self_help_controller );
 			$rest_self_help_controller->register_routes();
 
-			require_once __DIR__ . 'classes/class-wc-rest-connect-service-data-refresh-controller.php' ) );
+			require_once __DIR__ . '/classes/class-wc-rest-connect-service-data-refresh-controller.php';
```

Comment:
After merging from `develop`, I updated this line for phpunit and the web client to run as before

Reply:
😓 There was some merge conflict madness and this should be fixed with this PR! https://github.com/Automattic/woocommerce-services/pull/2350

Reply:
Cool I merged the latest `develop` to PR branch! I didn't get any merge conflicts 😄